src/test/TestObject.java

```diff
@@ -221,6 +222,23 @@ public static String getLastMethodOutput() {
 	}
 
 	/**
+	 * Gets everything the tested class printed through {@code Terminal.printLine} during the last method run, grouped
+	 * per <i>input</i> command. This means: If {@code n} Strings have been added for input through
+	 * {@link #setNextMethodCallInput(String[])}¹, this method's returned array will consist of {@code n+1} String
+	 * arrays. The first of these arrays will consist of the output printed <i>before</i> the tested class read the
+	 * first command. From there, the {@code i}th String array will consist of the output printed after the tested class
+	 * read the {@code i}th command. Each String in the String arrays corresponds to one call to
```

Comment:
Depends on your definition. But groupedOutput[1] will be the output of the first command. (Becaus groupedOutput [0] was before the first command).